compiler/src/dmd/common/bitfields.d

```diff
@@ -10,14 +10,18 @@
  */
 module dmd.common.bitfields;
 
+//version = Has_Bitfields; // does not work (yet) because hashOf doesn't work on bitfields
```

Comment:
What is this version `Has_Bitfields`, is it documented somewhere?

Reply:
`Has_Bitfields` is a potential version that could be defined instead of `Debugger_friendly` to use the builtin bitfields without the space penalty, but as noted, the current implementation of `hashOf` tries to take the address of the fields and causes the build to fail.